src/store/Dashboard.js

```diff
@@ -0,0 +1,263 @@
+import Vue from 'vue'
+import _ from 'lodash'
+import cryptoCompareUtil from 'util/cryptoApi-axios-util'
+
+const types = _([
+  'LOAD_DASHBOARD',
+  'GET_PRICE_BY_FILTER',
+  'GET_PORTFOLIO_HISTORY'
+]).chain()
+  .flatMap(x => [x + '_REQUEST', x + '_SUCCESS', x + '_FAILURE'])
+  .concat([
+    'RESET',
+    'GET_PORTFOLIO_FULL_PRICE',
+    'GET_PORTFOLIO_PRICE_LIST',
+    'GET_PORTFOLIO_PRICE_PERCENTAGE',
+    'SELECT_CHART_FILTER',
+    'SELECT_CHART_CRYPTO'
+  ])
+  .map(x => [x, x])
+  .fromPairs()
+  .value()
+
+const convertData = (obj, wallets) => {
+  const getEqTime = (t) => {
+    return _(obj)
+      .filter(c => c.data.length)
+      .map(c => ({
+        asset: c.asset,
+        value: _.find(c.data, d => d.time === t)
+      }))
+      .value()
+  }
+  const getSum = (d) => {
+    return d.reduce((p, n) => {
+      const crypto = wallets.find(c => c.asset === n.asset)
+      return p + (n.value.close * crypto.amount)
+    }, 0)
+  }
+  return _(obj[0].data)
```

Comment:
I didn't find it at first, but it seems that sometimes `obj` can have different `data` for each item.

![electron](https://user-images.githubusercontent.com/1365915/42677379-85f94de8-86b6-11e8-8694-20f6d0416dfb.png)

In this case, it seems cryptoCompare API returns empty arrays for Doge and EOS with different reasons:
```
// https://min-api.cryptocompare.com/data/histoday?fsym=Doge&tsym=RUB&limit=30
{"Response":"Error","Message":"There is no data for the symbol Doge .","Type":1,"Aggregated":false,"Data":[]}
```
```
// https://min-api.cryptocompare.com/data/histoday?fsym=EOS&tsym=RUB&limit=30
{"Response":"Success","Type":100,"Aggregated":false,"Data":[],"FirstValueInArray":true}
```

Maybe we should correct timestamps in another way. 🤔

Reply:
Now variable `timeKey` should get first item that has `time` key
![carbon 1](https://user-images.githubusercontent.com/16295803/42688381-b2b99ed0-86a4-11e8-8e06-bc9cc7f0ddfb.png)